internal/cli/atlas/customdbroles/update_test.go

```diff
@@ -73,3 +73,149 @@ func TestUpdateBuilder(t *testing.T) {
 		[]string{flag.ProjectID, flag.Output, flag.Privilege, flag.InheritedRole, flag.Append},
 	)
 }
+
+func Test_appendActions(t *testing.T) {
+	type args struct {
+		existingActions []mongodbatlas.Action
+		newActions      []mongodbatlas.Action
+	}
+	tests := []struct {
+		name string
+		args args
+		want []mongodbatlas.Action
+	}{
+		{
+			name: "empty",
+			args: args{
+				existingActions: []mongodbatlas.Action{},
+				newActions:      []mongodbatlas.Action{},
+			},
+			want: []mongodbatlas.Action{},
+		},
+		{
+			name: "no new actions",
+			args: args{
+				existingActions: []mongodbatlas.Action{
+					{
+						Action: "TEST",
+						Resources: []mongodbatlas.Resource{
+							{
+								Collection: "test",
+								Db:         "test",
+							},
+						},
+					},
+				},
+				newActions: []mongodbatlas.Action{},
+			},
+			want: []mongodbatlas.Action{
+				{
+					Action: "TEST",
+					Resources: []mongodbatlas.Resource{
+						{
+							Collection: "test",
+							Db:         "test",
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "different actions",
+			args: args{
+				existingActions: []mongodbatlas.Action{
+					{
+						Action: "TEST",
+						Resources: []mongodbatlas.Resource{
+							{
+								Collection: "test",
+								Db:         "test",
+							},
+						},
+					},
+				},
+				newActions: []mongodbatlas.Action{
+					{
+						Action: "NEW",
+						Resources: []mongodbatlas.Resource{
+							{
+								Collection: "test",
+								Db:         "test",
+							},
+						},
+					},
+				},
+			},
+			want: []mongodbatlas.Action{
+				{
+					Action: "TEST",
+					Resources: []mongodbatlas.Resource{
+						{
+							Collection: "test",
+							Db:         "test",
+						},
+					},
+				},
+				{
+					Action: "NEW",
+					Resources: []mongodbatlas.Resource{
+						{
+							Collection: "test",
+							Db:         "test",
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "merge",
+			args: args{
+				existingActions: []mongodbatlas.Action{
+					{
+						Action: "TEST",
+						Resources: []mongodbatlas.Resource{
+							{
+								Collection: "test",
+								Db:         "test2",
+							},
+						},
+					},
+				},
+				newActions: []mongodbatlas.Action{
+					{
+						Action: "TEST",
+						Resources: []mongodbatlas.Resource{
+							{
+								Collection: "test",
+								Db:         "test",
+							},
+						},
+					},
+				},
+			},
+			want: []mongodbatlas.Action{
+				{
+					Action: "TEST",
+					Resources: []mongodbatlas.Resource{
+						{
+							Collection: "test",
+							Db:         "test",
+						},
+						{
+							Collection: "test",
+							Db:         "test2",
+						},
+					},
+				},
+			},
+		},
+	}
```

Comment:
[q] Can we provide a scenario where we have more than 2 actions that are the same? `FIND@db1.collection1,FIND@db2.collection2,FIND@db3.collection3`